.gitignore

```diff
@@ -19,3 +19,4 @@ styleguide/source/styleguide/*
 styleguide/vendor/*
 styleguide/node_modules/*
 styleguide/.publish
+styleguide/tests/backstop/backstop_data/*
```

Comment:
@mrspete I think this was added when I ran a git rebase on this branch from develop. This would explain why we are seeing diffs that are outside the scope of this story.